.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The idea files should not be committed on the remote repository, as these are ide related files. Please ignore all the .idea files or ignore the .idea folder by git ignore command.